app/lib/custom_devise_token_generator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "openssl"
+
+class CustomDeviseTokenGenerator < Devise::TokenGenerator
```

Comment:
```suggestion
class ShortDeviseTokenGenerator < Devise::TokenGenerator
```
(On pourrait presque le mettre dans un namespace “Lapin”).

Reply:
Pour le coup j'aurais laissé ce nom là car cette classe est instanciée comme le token generator de toute l'app, même si elle vise principalement à ajouter la méthode generate_short_token (et si on prend en compte ta modif il faut modifier le nom du fichier aussi)

Reply:
ok, ça me va.